examples/src/main/scala/feature/GeoCrunch.scala

```diff
@@ -39,7 +39,7 @@ trait GeoCrunch {
   ) = {
     val it = fc.features()
     try {
-      while (it.hasNext) try {
```

Comment:
Yep it's totally superfluous, possibly a leftover from a poorly-executed refactoring.  I agree with the decision to remove it.